<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_loadProject</class>
 <widget class="QDialog" name="Dialog_loadProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load Project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_LoadProject_ProjectAuthor">
       <property name="text">
        <string>Project Author</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_LoadProject_ProjectName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_LoadProject_ProjectDescription">
       <property name="text">
        <string>Project Description</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_LoadProject_ProjectName">
       <property name="text">
        <string>Select Project Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_LoadProject_ProjectAuthor"/>
     </item>
     <item row="3" column="1">
      <widget class="QDialogButtonBox" name="buttonBox_LoadProject">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="textEdit_LoadProject_ProjectDescription"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
